src/Parallel/dominatingset/minimal_dom_set.jl

```diff
@@ -1,5 +1,5 @@
 """
-    dominating_set(g, reps, MinimalDominatingSet(); parallel=:threads, seed=-1)
+    dominating_set(g, reps, MinimalDominatingSet(); parallel=:threads, kw...)
```

Comment:
I think we should mention the role of `rng` (and potentially also `seed` altough we want to phase that one out)  in the documentation.

Reply:
I am going to restore that. I've tried to be consistent with all the docstrings such as https://github.com/JuliaGraphs/Graphs.jl/blob/903f309039fa71ae2364aaec2158939f76cf5ccf/src/dominatingset/minimal_dom_set.jl#L6, but I might have missed some.

Reply:
This is fixed.